examples/example_minimal.py

```diff
@@ -76,7 +76,7 @@ def objective(individual):
     "n_outputs": 1,
     "n_columns": 12,
     "n_rows": 1,
-    "levels_back": 5,
+    "levels_back": None,
```

Comment:
please undo this change again before merging

Reply:
Yes, I will do. Should I add an example showing the use of `reorder`?
Or should I integrate it within an existing example (not sure it fits somewhere...)?

Reply:
i think it would be good idea to provide a separate example, maybe comparing the time-to-solution with and without reordering

Reply:
I added an example which compares reordered and non-reordered history on the same target function as in minimal example. Please tell me if this is suited.